students/Cracolici_Jon/lesson03/JonCracolici_L3_Mailroom1.py

```diff
@@ -0,0 +1,105 @@
+#Jon Cracolici
+#Lesson 03 Mailroom 1
+#UW Python Cert
+
+#Below are the functions my program uses
+
+def initializedatabase():
+    """This function intializes the database when called."""
+    d1 = ['William Gates, III', 653784.49, 2, 326892.24]
+    d2 = ['Mark Zuckerberg ', 16396.10 , 3, 5465.37]
+    d3 = ['Jeff Bezos', 877.33, 1, 877.33]
+    d4 = ['Paul Allen', 708.42, 3, 236.14]
+    dB = [d1, d2, d3, d4]
+    return dB
+#	
+def newdonor(name, dB):
+    """Creates a new donor in the database and requests donation amount."""
+    donation = float(input("How much did {} donate?".format(name)))
+    newdonor = [name, donation, 1, donation]
+    dB.append(newdonor)
+    return name, donation
+#
+def updatedonor(name, dB):
+    """Updates an existing donor in the database with a new donation."""
+    for item in dB:
+        if item[0] == str(name):
+            newdonation = float(input("How much did {} donate?".format(name)))
+            print(item)
+            print(type(item[1]))
+            item[1] += float(newdonation)
+            item[2] += 1
+            item[3] = item[1]/item[2]
```

Comment:
add a `break` here so that you don't continue looping after you've found your donor.